crime-means-assessment/build.gradle

```diff
@@ -67,12 +68,25 @@ dependencies {
     testImplementation "org.springframework.security:spring-security-test"
     testImplementation "com.squareup.okhttp3:okhttp:$versions.okhttpVersion"
     testImplementation "org.springframework.cloud:spring-cloud-starter-contract-stub-runner:$versions.springCloudStubRunnerVersion"
+
+    //  cucumber
+//    testImplementation "org.projectlombok:lombok:1.18.26"
+    testImplementation "org.junit.platform:junit-platform-suite"
+    testImplementation "io.cucumber:cucumber-java:$versions.cucumberVersion"
+    testImplementation("io.cucumber:cucumber-java8:$versions.cucumberVersion")
+    testImplementation "io.cucumber:cucumber-junit:$versions.cucumberVersion"
+    testImplementation "io.cucumber:cucumber-junit-platform-engine:$versions.cucumberVersion"
+    testImplementation "io.cucumber:cucumber-spring:$versions.cucumberVersion"
```

Comment:
Quite a few of these dependencies can be removed, I think we just need:

- `cucumber-junit`
- `cucumber-spring`
- `cucumber-java`